src/main/resources/story/web/visualCheck.story

```diff
@@ -0,0 +1,4 @@
+Scenario: Visual check Log-in
```

Comment:
Login page is not displayed on baseline

Reply:
Also, if you're going to cover login page with visual checks, maybe it would be better to add this steop to existing login/logout story

Reply:
Comment is still relevant. I think it would be better to add visual check right into Login story. There is no need for seperate story with one scenario, partially copying already exisitng scenario

Reply:
Last repot (27/01/2023) [allure.zip](https://github.com/Yanina-Matuzava/test-project-matuzava/files/10521701/allure.zip)